templates/notices/error.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Show error messages
+ *
+ * This template can be overridden by copying it to yourtheme/woocommerce/notices/error.php.
+ *
+ * HOWEVER, on occasion WooCommerce will need to update template files and you
+ * (the theme developer) will need to copy the new files to your theme to
+ * maintain compatibility. We try to do this as little as possible, but it does
+ * happen. When this occurs the version of the template file will be bumped and
+ * the readme will list any important changes.
+ *
+ * @see     https://docs.woocommerce.com/document/template-structure/
+ * @package WooCommerce\Templates
+ * @version 3.9.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! $notices ) {
+	return;
+}
+
+$notices[] = $notices[0];
```

Comment:
Got bamboozled by this one 😂

So we're appending the first item to the array, by accessing it without checking, it means it exists and $multiple will always be true because the min count() is 2 right? What am I missing here? I must have to be reading something wrong, sorry. Or is it forgotten code to test for multiple notices?

Reply:
Sorry this was debugging code, I've removed it.